### ISSUE TITLE
image not refreshed correctly

### ISSUE THREAD

**Issue by williamwongys** [scalableminds/chatroom#140]
I passed updated image from custom action to chatroom, but every time the same image is being displayed instead of the new image.  I have checked the png file, it is the latest.  I suspect chatroom caches the image instead of retrieving from the disk again.  Is there any way I could force retrieval of the image again?

**Comment by hotzenklotz** [comment]
I don't think we cache anything. Have you ruled out and browser caches? Or server-side caches.
